docs/src/Video.doc.js

```diff
@@ -92,6 +91,11 @@ card(
         type: '({ fullscreen: boolean }) => void',
         description: 'Sent when the video full screen status changes',
       },
+      {
+        name: 'onLoadedChange',
```

Comment:
So I wanted to maintain an API where `onXChange` returns `x` in the callback because the `x` value internally changed. Its not controlled, you're right, and kind of follows the `onFullscreenChange` pattern where it returns `fullscreen` value when it changes internally.
Not a huge fan of the name either but at least it conveys that when this callback is called it will return the time in seconds that is `loaded`. I'm really not sure why the native element chose `onProgress` because to me that almost implies progress playing the video, not loading it. Would `onProgressChange` returning `progress` or `onBufferedChanged` returning `buffered` make more sense? Open to discussion here